lib/metasploit/framework/data_service/remote/http/response_data_helper.rb

```diff
@@ -47,17 +48,18 @@ def json_to_mdm_object(response_wrapper, mdm_class, returns_on_error = nil)
     if response_wrapper.expected
       begin
         body = process_response(response_wrapper)
-        unless body.nil? || body.empty?
-          parsed_body = Array.wrap(JSON.parse(body))
+        if !body.nil? || !body.empty?
```

Comment:
Minor: I'm happy to see this change back as I think `unless` is best with a single condition. See line 29 above as you could make a similar modification.